src/TestSuite/TestCase.php

```diff
@@ -804,4 +804,21 @@ public static function setAppNamespace($appNamespace = 'TestApp')
     {
         Configure::write('App.namespace', $appNamespace);
     }
+
+    /**
+     * Gets fixtures.
+     *
+     * @return array
+     */
+    public function getFixtures()
+    {
+        if (!isset($this->fixtures)) {
+            return [];
+        }
+        if (is_string($this->fixtures)) {
```

Comment:
Why does this have to be a string?  In 4.x `$fixtures` property is already declared / initialized as array.

Reply:
It is moved from FixtureManager::fixturize() - https://github.com/cakephp/cakephp/pull/13586/files#diff-00c99fa549dd6a8e04203fff9bcc684aL100.

Reply:
That part will be taken care of follow up https://github.com/cakephp/cakephp/pull/13593